modules/sdk/src/test/scala/org/tessellation/sdk/infrastructure/trust/TrustRatingCsvLoaderSuite.scala

```diff
@@ -0,0 +1,19 @@
+package org.tessellation.sdk.infrastructure.trust
+
+import cats.effect.IO
+
+import fs2.io.file.Path
+import weaver._
+import weaver.scalacheck._
+
+object TrustRatingCsvLoaderSuite extends SimpleIOSuite with Checkers {
+  test("load trust ratings from csv") {
+    val testFileLocation = getClass.getResource("/ratings.sample.csv").getPath
+    val inFile = Path(testFileLocation)
+
+    for {
+      csv <- TrustRatingCsvLoader.make[IO].load(inFile)
+      _ = println(csv)
+    } yield success
```

Comment:
println shouldn't be a part of a test. you specified a sample rating, so the test assertion should expect that specific rating to be red

Reply:
Replaced with `expect.eql`.